packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -43,6 +43,11 @@ contract TaikoL1 is
     TaikoData.State public state;
     uint256[100] private __gap;
 
+    modifier whenProvingNotPaused() {
```

Comment:
Just some simplification for the verification logic when proving is paused or nothing needs to be verified, nothing important.